docs/quickstart.rst

```diff
@@ -3,361 +3,556 @@
 Quickstart
 ***************
 
-This tutorial gets you started with the Inmanta orchestration tool.
-
 Inmanta is intended to manage complex infrastructures, often in the cloud or other virtualized environments.
-In this guide, we go for a less complex setup: install the Drupal CMS on two VM-like containers.
-First, we use Docker to set up a basic environment with two empty VM-like containers, an Inmanta server and a postgres server used by inmanta as a database.
-Then, we use Inmanta to install Drupal on these VM-like containers.
+In this guide, we go for a less complex setup: Installing `containerlab <https://containerlab.dev/>`_ and configuring `SR Linux <https://learn.srlinux.dev/>`_ containers using **Inmanta service orchestrator** and ``gNMI``.
+
+
+1. First, we use `Containerlab` to spin-up Inmanta server and its PostgreSQL database, then three `SR Linux` containers, connected in a CLOS like topology
+2. After that, we configure IP addresses and OSPF on them using **Inmanta**.
 
 .. note::
 
-    This is meant to get an example Inmanta environment set up and running quickly to experiment with.
+    This guide is meant to quickly set up an Inmanta LAB environment to experiment with.
     It is not recommended to run this setup in production, as it might lead to instabilities in the long term.
 
-.. _qsetup:
 
-Setting up the tutorial
-_________________________
+Prerequisites
+----------------------------
+
+``Docker``, ``Containerlab`` and ``Inmanta`` need to be installed on your machine and our ``SR Linux`` repository has to be cloned in order to proceed. Please make sure to follow the links below to that end.
+
+1. `Install Docker <https://docs.docker.com/install/>`_.
+2. `Install Containerlab <https://containerlab.dev/install/>`_.
+3. Prepare a development environment by creating a `python virtual environment` and installing Inmanta:
+
+   .. code-block:: sh
+
+       mkdir -p ~/.virtualenvs
+       python3 -m venv ~/.virtualenvs/srlinux
+       source ~/.virtualenvs/srlinux/bin/activate
+       pip install inmanta-core
+
+4. Clone the `SR Linux examples <https://github.com/inmanta/examples/tree/master/Networking/SR%20Linux>`_ repository:
+
+   .. code-block:: sh
+
+       git clone https://github.com/inmanta/examples.git
+
+
+The cloned repository contains a **project.yml**, which looks like this:
+
+    .. code-block:: yaml
+
+        name: SR Linux Examples
+        description: Provides examples for the SR Linux module
+        author: Inmanta
+        author_email: code@inmanta.com
+        license: ASL 2.0
+        copyright: 2022 Inmanta
+        modulepath: libs
+        downloadpath: libs
+        repo:
+            - https://github.com/inmanta/
```

Comment:
TODO: we need to fix this to provide a repo that contains the srlinux and yang modules.